### PR TITLE
Added the sender in smite.

### PR DESCRIPTION
**Pull request by ghost**
Added sender in smite. (private)

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java [hunk 1]
```diff
@@ -15,74 +15,47 @@
...
+    public static void smite(final Player player, final String reason, String sender) {
         FUtil.bcastMsg(player.getName() + " has been a naughty, naughty boy.", ChatColor.RED);
-
-        if (reason != null)
-        {
+        if (reason != null && sender != null) {
             FUtil.bcastMsg("  Reason: " + reason, ChatColor.RED);
         }
-
-        // Deop
```
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java]: Why did you remove this? How will other noobs understand what each thing does?

**Comment by Wild1145** [comment]
What issue does this resolve?

**Comment by marcocorriero** [comment]
@Wild1145 it adds smite saying the admin who smited op , and if the op wanna know who smited, then this is the fix

**Comment by Wild1145** [comment]
@marcocorriero Again, what issue number does this resolve

**Comment by marcocorriero** [comment]
#1776

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java (hunk 1 above)
Comment (DUFCLiam) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java]: The noobs will not be reading code.

**Comment by Wild1145** [comment]
Please check through and re-format to comply with the proper contributing guidelines.

**Comment by marcocorriero** [comment]
@Wild1145 i did a test on marcofreedom. Works fine.

**Comment by marcocorriero** [comment]
Fail, wrong button lol

**Comment by Wild1145** [comment]
@marcocorriero It still needs changes. Please read the entire of my comments before posting in the future.
